Watching for file changes with StatReloader
Not Found: /api/auth/token/
[04/Oct/2025 18:03:37] "POST /api/auth/token/ HTTP/1.1" 404 4261
[04/Oct/2025 18:03:37] "POST /api/token/ HTTP/1.1" 200 489
[04/Oct/2025 18:03:37] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:03:38] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:03:38] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:03:41] "OPTIONS /api/predict/ HTTP/1.1" 200 0
[04/Oct/2025 18:03:41] "GET /api/predict/ HTTP/1.1" 200 1276
[04/Oct/2025 18:03:41] "OPTIONS /api/execute_trade/ HTTP/1.1" 200 0
[04/Oct/2025 18:03:42] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 18:03:42] "OPTIONS /api/mint_energy/ HTTP/1.1" 200 0
[04/Oct/2025 18:03:42] "POST /api/mint_energy/ HTTP/1.1" 200 148
[04/Oct/2025 18:04:26] "GET /api/predict/ HTTP/1.1" 200 1267
[04/Oct/2025 18:04:26] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 18:04:26] "POST /api/mint_energy/ HTTP/1.1" 200 148
[04/Oct/2025 18:10:45] "GET /api/predict/ HTTP/1.1" 200 1276
[04/Oct/2025 18:10:45] "GET /api/predict/ HTTP/1.1" 200 1277
[04/Oct/2025 18:11:59] "GET /api/predict/ HTTP/1.1" 200 1277
[04/Oct/2025 18:12:04] "GET /api/predict/ HTTP/1.1" 200 1277
[04/Oct/2025 18:12:13] "POST /api/execute_trade/ HTTP/1.1" 200 492
[04/Oct/2025 18:12:30] "POST /api/mint_energy/ HTTP/1.1" 200 127
[04/Oct/2025 18:13:59] "GET /api/predict/ HTTP/1.1" 200 1278
[04/Oct/2025 18:14:27] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:14:27] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:14:27] "GET /api/predict/ HTTP/1.1" 200 1277
[04/Oct/2025 18:14:27] "GET /api/predict/ HTTP/1.1" 200 1266
[04/Oct/2025 18:14:49] "GET /api/predict/ HTTP/1.1" 200 1269
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:02:36
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.13/threading.py", line 1043, in _bootstrap_inner
    self.run()
    ~~~~~~~~^^
  File "/usr/lib/python3.13/threading.py", line 994, in run
    self._target(*self._args, **self._kwargs)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
    ~~^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/management/commands/runserver.py", line 134, in inner_run
    self.check(**check_kwargs)
    ~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/management/base.py", line 492, in check
    all_issues = checks.run_checks(
        app_configs=app_configs,
    ...<2 lines>...
        databases=databases,
    )
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/checks/registry.py", line 89, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/checks/urls.py", line 16, in check_url_config
    return check_resolver(resolver)
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/checks/urls.py", line 26, in check_resolver
    return check_method()
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/resolvers.py", line 531, in check
    for pattern in self.url_patterns:
                   ^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/resolvers.py", line 718, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
                       ^^^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/resolvers.py", line 711, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/michael/Desktop/p2p/p2p-energy-backend/p2p_energy_backend/urls.py", line 9, in <module>
    path("api/", include("trading.urls")),
                 ~~~~~~~^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/conf.py", line 39, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/michael/Desktop/p2p/p2p-energy-backend/trading/urls.py", line 2, in <module>
    from .views import predict, execute_trade, mint_energy
  File "/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py", line 40
    data = request.body
    ^^^^
IndentationError: expected an indented block after 'try' statement on line 39
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.13/threading.py", line 1043, in _bootstrap_inner
    self.run()
    ~~~~~~~~^^
  File "/usr/lib/python3.13/threading.py", line 994, in run
    self._target(*self._args, **self._kwargs)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
    ~~^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/management/commands/runserver.py", line 134, in inner_run
    self.check(**check_kwargs)
    ~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/management/base.py", line 492, in check
    all_issues = checks.run_checks(
        app_configs=app_configs,
    ...<2 lines>...
        databases=databases,
    )
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/checks/registry.py", line 89, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/checks/urls.py", line 16, in check_url_config
    return check_resolver(resolver)
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/core/checks/urls.py", line 26, in check_resolver
    return check_method()
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/resolvers.py", line 531, in check
    for pattern in self.url_patterns:
                   ^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/resolvers.py", line 718, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
                       ^^^^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ~~~~~~~~~^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/resolvers.py", line 711, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/michael/Desktop/p2p/p2p-energy-backend/p2p_energy_backend/urls.py", line 9, in <module>
    path("api/", include("trading.urls")),
                 ~~~~~~~^^^^^^^^^^^^^^^^
  File "/home/michael/Desktop/p2p/p2p-energy-backend/.venv/lib/python3.13/site-packages/django/urls/conf.py", line 39, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/home/michael/Desktop/p2p/p2p-energy-backend/trading/urls.py", line 2, in <module>
    from .views import predict, execute_trade, mint_energy, marketplace, buy_listing
  File "/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py", line 40
    data = request.body
    ^^^^
IndentationError: expected an indented block after 'try' statement on line 39
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

Watching for file changes with StatReloader
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:36:57
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
/home/michael/Desktop/p2p/p2p-energy-backend/trading/urls.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:37:07
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
[04/Oct/2025 18:38:01] "GET /api/account/balance/ HTTP/1.1" 200 22
[04/Oct/2025 18:39:10] "POST /api/buy_listing/ HTTP/1.1" 200 328
[04/Oct/2025 18:39:19] "GET /api/account/balance/?household_id=HH_UI_001 HTTP/1.1" 200 22
[04/Oct/2025 18:39:59] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 18:39:59] "POST /api/mint_energy/ HTTP/1.1" 200 144
[04/Oct/2025 18:39:59] "OPTIONS /api/account/balance/ HTTP/1.1" 200 0
[04/Oct/2025 18:39:59] "GET /api/account/balance/ HTTP/1.1" 200 22
Unauthorized: /api/me/
[04/Oct/2025 18:40:05] "GET /api/me/ HTTP/1.1" 401 172
Unauthorized: /api/me/
[04/Oct/2025 18:40:05] "GET /api/me/ HTTP/1.1" 401 172
[04/Oct/2025 18:40:05] "OPTIONS /api/auth/token/refresh/ HTTP/1.1" 200 0
Not Found: /api/auth/token/refresh/
[04/Oct/2025 18:40:05] "POST /api/auth/token/refresh/ HTTP/1.1" 404 5100
[04/Oct/2025 18:40:05] "OPTIONS /api/token/refresh/ HTTP/1.1" 200 0
[04/Oct/2025 18:40:05] "POST /api/token/refresh/ HTTP/1.1" 200 244
[04/Oct/2025 18:40:05] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:40:05] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:40:06] "GET /api/predict/ HTTP/1.1" 200 1278
[04/Oct/2025 18:40:06] "GET /api/predict/ HTTP/1.1" 200 1278
[04/Oct/2025 18:40:12] "OPTIONS /api/marketplace/ HTTP/1.1" 200 0
[04/Oct/2025 18:40:12] "OPTIONS /api/marketplace/ HTTP/1.1" 200 0
Not Found: /api/marketplace/
[04/Oct/2025 18:40:12] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:40:12] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:40:16] "OPTIONS /api/buy_listing/ HTTP/1.1" 200 0
[04/Oct/2025 18:40:16] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:40:18] "POST /api/buy_listing/ HTTP/1.1" 200 327
Not Found: /api/marketplace/
[04/Oct/2025 18:40:41] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:40:41] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:40:43] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:40:44] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:40:45] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:40:46] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:40:48] "GET /api/predict/ HTTP/1.1" 200 1278
[04/Oct/2025 18:40:48] "GET /api/predict/ HTTP/1.1" 200 1268
[04/Oct/2025 18:41:01] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 18:41:01] "POST /api/mint_energy/ HTTP/1.1" 200 144
[04/Oct/2025 18:41:01] "GET /api/account/balance/ HTTP/1.1" 200 22
[04/Oct/2025 18:45:35] "GET /api/predict/ HTTP/1.1" 200 1278
[04/Oct/2025 18:45:35] "GET /api/predict/ HTTP/1.1" 200 1278
[04/Oct/2025 18:45:43] "GET /api/predict/ HTTP/1.1" 200 1278
Bad Request: /api/execute_trade/
[04/Oct/2025 18:48:05] "POST /api/execute_trade/ HTTP/1.1" 400 54
Not Found: /api/marketplace/
[04/Oct/2025 18:48:15] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:48:15] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:48:18] "POST /api/buy_listing/ HTTP/1.1" 200 327
Not Found: /api/marketplace/
[04/Oct/2025 18:48:27] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:48:27] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:48:35] "GET /api/predict/ HTTP/1.1" 200 1269
[04/Oct/2025 18:48:35] "GET /api/predict/ HTTP/1.1" 200 1269
Not Found: /api/marketplace/
[04/Oct/2025 18:48:42] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:48:42] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:48:44] "GET /api/predict/ HTTP/1.1" 200 1269
[04/Oct/2025 18:48:44] "GET /api/predict/ HTTP/1.1" 200 1269
Not Found: /api/marketplace/
[04/Oct/2025 18:48:47] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:48:47] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:48:54] "GET /api/predict/ HTTP/1.1" 200 1269
[04/Oct/2025 18:48:54] "GET /api/predict/ HTTP/1.1" 200 1269
Bad Request: /api/execute_trade/
[04/Oct/2025 18:48:56] "POST /api/execute_trade/ HTTP/1.1" 400 54
[04/Oct/2025 18:50:54] "GET /api/predict/ HTTP/1.1" 200 1268
Bad Request: /api/execute_trade/
[04/Oct/2025 18:51:33] "POST /api/execute_trade/ HTTP/1.1" 400 54
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:37:23
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
Bad Request: /api/execute_trade/
[04/Oct/2025 18:51:53] "POST /api/execute_trade/ HTTP/1.1" 400 52
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:51:47
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
[04/Oct/2025 18:53:54] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 18:54:27] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:54:27] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 18:54:27] "GET /api/predict/ HTTP/1.1" 200 1268
[04/Oct/2025 18:54:27] "GET /api/predict/ HTTP/1.1" 200 1268
[04/Oct/2025 18:54:29] "POST /api/execute_trade/ HTTP/1.1" 200 490
[04/Oct/2025 18:54:29] "POST /api/mint_energy/ HTTP/1.1" 200 144
[04/Oct/2025 18:54:29] "GET /api/account/balance/ HTTP/1.1" 200 22
Not Found: /api/marketplace/
[04/Oct/2025 18:54:40] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:54:40] "GET /api/marketplace/ HTTP/1.1" 404 52
[04/Oct/2025 18:54:42] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:54:44] "POST /api/buy_listing/ HTTP/1.1" 200 327
[04/Oct/2025 18:55:36] "GET /api/predict/ HTTP/1.1" 200 1268
[04/Oct/2025 18:55:36] "GET /api/predict/ HTTP/1.1" 200 1268
[04/Oct/2025 18:55:37] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 18:55:37] "POST /api/mint_energy/ HTTP/1.1" 200 144
[04/Oct/2025 18:55:38] "GET /api/account/balance/ HTTP/1.1" 200 22
Not Found: /api/marketplace/
[04/Oct/2025 18:57:15] "GET /api/marketplace/ HTTP/1.1" 404 52
Not Found: /api/marketplace/
[04/Oct/2025 18:57:15] "GET /api/marketplace/ HTTP/1.1" 404 52
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:53:41
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:58:22
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:58:34
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
[04/Oct/2025 18:58:52] "GET /api/predict/?household=HH_UI_001 HTTP/1.1" 200 1227
[04/Oct/2025 19:01:21] "GET /api/predict/ HTTP/1.1" 200 1249
[04/Oct/2025 19:01:22] "GET /api/predict/ HTTP/1.1" 200 1249
[04/Oct/2025 19:01:23] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 19:01:23] "GET /api/me/ HTTP/1.1" 200 116
[04/Oct/2025 19:01:23] "GET /api/predict/ HTTP/1.1" 200 1249
[04/Oct/2025 19:01:24] "GET /api/predict/ HTTP/1.1" 200 1249
[04/Oct/2025 19:01:25] "POST /api/execute_trade/ HTTP/1.1" 200 491
[04/Oct/2025 19:01:25] "POST /api/mint_energy/ HTTP/1.1" 200 235
[04/Oct/2025 19:01:25] "GET /api/account/balance/ HTTP/1.1" 200 22
Unauthorized: /api/me/
[04/Oct/2025 19:03:49] "GET /api/me/ HTTP/1.1" 401 4988
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3796
[04/Oct/2025 19:03:49] "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3426
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1793
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/jquery-3.7.1.min.js HTTP/1.1" 200 87533
[04/Oct/2025 19:03:49] "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
[04/Oct/2025 19:03:49] "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/load-ajax-form.js HTTP/1.1" 200 59
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
[04/Oct/2025 19:03:49] "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
[04/Oct/2025 19:03:49] "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
[04/Oct/2025 19:03:49] "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
Not Found: /favicon.ico
[04/Oct/2025 19:03:49] "GET /favicon.ico HTTP/1.1" 404 2683
[04/Oct/2025 19:03:55] "GET /api/predict/ HTTP/1.1" 200 1249
[04/Oct/2025 19:04:06] "POST /api/buy_listing/ HTTP/1.1" 200 328
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 18:58:44
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
[04/Oct/2025 19:04:26] "POST /api/buy_listing/ HTTP/1.1" 200 328
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 19:04:18
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 19:04:51
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
/home/michael/Desktop/p2p/p2p-energy-backend/trading/urls.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 19:05:02
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
[04/Oct/2025 19:05:26] "POST /api/mpesa/stk_push/ HTTP/1.1" 200 170
[04/Oct/2025 19:05:32] "POST /api/mpesa/callback/ HTTP/1.1" 200 60
/home/michael/Desktop/p2p/p2p-energy-backend/trading/views.py changed, reloading.
Performing system checks...

System check identified no issues (0 silenced).
October 04, 2025 - 19:05:16
Django version 5.2.7, using settings 'p2p_energy_backend.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

WARNING: This is a development server. Do not use it in a production setting. Use a production WSGI or ASGI server instead.
For more information on production servers see: https://docs.djangoproject.com/en/5.2/howto/deployment/
Watching for file changes with StatReloader
[04/Oct/2025 19:11:28] "POST /api/mpesa/stk_push/ HTTP/1.1" 200 193
